src/provider/qti.js

```diff
@@ -614,6 +614,9 @@ var qtiProvider = {
                     self.trigger('enablenav');
                     reject(err);
                 })
+                .on('itemrender.error', function (err) {
```

Comment:
the other war around, please use an namespace on `error` : 
```
on('error.itemrender`, function(err){
```

Reply:
Hello. I just checked and if I will trigger `error.itemrender` then that event will be caught by 
`.on('error.itemrender'` and by `.on('error'`.

Reply:
I'd suggest then to add the property `unrecoverable` to the `Error`  : 
```
const err = new Error('Item rendering issue');
err.unrecoverable = true;
runner.trigger('error', err);
```

then inside the test runner implementation : 
```
.on('error', err => {
  if(err && err.unrecoverable){
     runner.pause();
  } else {
      //...
  }
})
```

we already are using this formalism for cancellable errors

Reply:
got it. Thanks a lot @krampstudio